src/Pages/MyPage/ProfileEdit.js

```diff
@@ -0,0 +1,164 @@
+import React, { useEffect, useState, useRef } from 'react';
+import './ProfileEdit.scss';
+
+const ProfileEdit = ({ setEditClicked, user }) => {
+  const accessToken = localStorage.getItem('TOKEN');
+  const fileInput = useRef(null);
+  const [imgFile, setImgFile] = useState({});
+  const [imageSrc, setImageSrc] = useState(user.profile_image);
+  const [input, setInput] = useState({
+    nickname: user.nickname,
+    age: user.age,
+    gender: user.gender,
+  });
+
+  const encodeFileToBase64 = fileBlob => {
+    const reader = new FileReader();
+    reader.readAsDataURL(fileBlob);
+    return new Promise(resolve => {
+      reader.onload = () => {
+        setImageSrc(reader.result);
+        resolve();
+      };
+    });
+  };
+
+  const handleImageInput = e => {
+    encodeFileToBase64(e.target.files[0]);
+    setImgFile(e.target.files[0]);
+  };
+
+  const handleInput = e => {
+    const { name, value } = e.target;
+    setInput({ ...input, [name]: value });
+  };
+
+  const profileEditForm = new FormData();
+  profileEditForm.append('profile_image', imgFile);
+  profileEditForm.append('nickname', input.nickname);
+  profileEditForm.append('gender', input.gender);
+  profileEditForm.append('age', input.age);
+
+  const profileEditSave = e => {
+    e.preventDefault();
+    fetch('http://192.168.215.82:3000/user/profile', {
+      method: 'PATCH',
+      headers: {
+        enctype: 'multipart/form-data',
+        authorization: accessToken,
+      },
+      body: profileEditForm,
+    })
+      .then(res => res.json())
+      .then(data => console.log(data))
+      .then(() => window.location.reload())
+      .then(() => setEditClicked(false));
```

Comment:
```suggestion
      .then(data => {
        if (data.message === 'success') {
          window.location.reload();
          setEditClicked(false);
        }
      })
```

fetch 에서 PATCH에 대한 데이터는 통신이 잘 됐는지 아닌지에 대한 메세지를 응답받을텐데
그 메세지를 이용해서 이러한 분기처리를 해주는 방식이 더 좋겠습니다!

data.message는 제가 임의로 작성한 값이니 백엔드에서 전달받는 값으로 구현해주세요!